propagation.md

```diff
@@ -1,73 +1,74 @@
 
 Let's briefly restate the "Why OpenTracing?" section of the [OpenTracing landing page](/):
 
 > By offering consistent, expressive, vendor-neutral APIs for popular platforms, OpenTracing makes it easy for developers to add (or switch) tracing implementations with an O(1) configuration change. OpenTracing also offers a lingua franca for OSS instrumentation and platform-specific tracing helper libraries.
 
-**Together, `Inject` and `Join` allow for inter-process trace propagation without tightly coupling the programmer to a particular OpenTracing implementation.** 
+**Together, `Inject` and `Extract` allow for inter-process trace propagation without tightly coupling the programmer to a particular OpenTracing implementation.** 
 
 ## Requirements for the OpenTracing propagation scheme
 
-For `Inject` and `Join` to be useful, all of the following must be true:
+For `Inject` and `Extract` to be useful, all of the following *must* be true:
 
 - Per the above, the [OpenTracing user](/use-cases/#stepping-back-who-is-opentracing-for) handling cross-process trace propagation must not need to write OpenTracing-implementation-specific code
 - OpenTracing implementations must not need special handlers for every known inter-process communication mechanism: that's far too much work, and it's not even well-defined
 - That said, the propagation mechanism should be extensible for optimizations
 
-## The basic approach: Inject, Join, and Carriers
+## The basic approach: Inject, Extract, and Carriers
 
-Any Span in a trace may be **Injected** into what OpenTracing refers to as a Carrier. A **Carrier** is an interface or data structure useful for inter-process communication (IPC); that is, the Carrier is something that "carries" the tracing state from one process to another. The OpenTracing specification includes two [required Carrier formats](#required-carriers), though [custom Carrier formats](#custom-carriers) are possible as well.
+Any SpanContext in a trace may be **Injected** into what OpenTracing refers to as a Carrier. A **Carrier** is an interface or data structure that's *useful* for inter-process communication (IPC); that is, the Carrier is something that "carries" the tracing state from one process to another. The OpenTracing specification includes two [required Carrier formats](#required-carriers), though [custom Carrier formats](#custom-carriers) are possible as well.
```

Comment:
I meant "useful" in that the literal data structure used for IPC is rarely going to be general enough... since we came up with that terminology, though, the "carrier" has become less of an object/struct and more of an abstract interface for reading/writing a certain type of data (e.g., raw bytes, key:val pairs, or http headers).

I want to sleep on this, but I can imagine `s/carrier/projector/` or something (the interfaces have started to look more like that lately anyway).